### PR TITLE
Support force display in opensim-viewer

### PR DESCRIPTION
**Pull request by aymanhab**
Add utilities that do the meaty part of the backend, converting various file formats trc, mot, c3d to gltf

### CODE REVIEW

**Comment by aymanhab** [comment]
[GRF_walk_1_25ms_0.zip](https://github.com/opensim-org/opensim-viewer/files/11562331/GRF_walk_1_25ms_0.zip)
Zip file has three gltf files resulting from converting trc, .mot and .c3d files

**Comment by AlbertoCasasOrtiz** [comment]
I've tested the files in the zip and I have some comments, but maybe this is what we expect:

- File `GRF_walk_1_25ms_0` time starts at instant 24.
- File `subject02_walk1_grf` only has one frame.
- File `Trimmed_Sprint_08550001` only has ground reaction force for one feet.

**Comment by AlbertoCasasOrtiz** [comment]
@aymanhab Regarding the python scripts, do you have trc, mot and c3d files that I can use for testing?

File: src/backend/convertC3D2Gltf.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+          shape = 'arrow'
+          unitConversionToMeters = .001
+          scaleData = True
+          if (forcesDataTable.hasTableMetaDataKey("Units")) :
+              unitString = forcesDataTable.getTableMetaDataString("Units")
+              if (unitString=="mm"):
+                  unitConversionToMeters = .001
+                  scaleData = True
+          else:
+              print("File has no Units specifications, NMS assumed.")
```
Comment (AlbertoCasasOrtiz) [inline src/backend/convertC3D2Gltf.py:72]: When this is integrated into the django backend, we could localize these strings.

**Review by AlbertoCasasOrtiz — COMMENTED** [review]
I have yet to test the scripts with my own files, but the code and the results you sent me in the zip file look good. (I left some comments about the zip file).

File: src/backend/convertC3D2Gltf.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    # have force data, need to verify this to be the case for OpenSim use cases
+    numMarkerFrames = markerDataTable.getNumRows()
+    if numMarkerFrames==0:
+        raise IndexError("Input file has no data", markerDataTable)
+    # instead of creating the GLTF2 structure from scratch and programmatically adding basic
+    # shapes (Sphere, Cube, brick, axes, arrows etc.) instead we load a file with these meshes
+    # baked in and use these meshes as needed. 
+    gltf = os2Gltf.initGltf()
+
+    osim.STOFileAdapterVec3.write(markerDataTable, 'c3dMarkerData.sto')
```
[the remark this answers is not in the document]
Comment (aymanhab) [reply src/backend/convertC3D2Gltf.py:47]: No this was a debugging line that should be removed. Thanks for the find.

**Review by AlbertoCasasOrtiz — APPROVED** [review]

**Comment by aymanhab** [comment]
Thanks @AlbertoCasasOrtiz much appreciated.

**Pull request merged**